api/v2beta1/helmrelease_types.go

```diff
@@ -949,6 +949,11 @@ type HelmReleaseStatus struct {
 	// +optional
 	LastAttemptedConfigDigest string `json:"lastAttemptedConfigDigest,omitempty"`
 
+	// LastAttemptedPostRenderersDigest is the digest for the post-renderers of
+	// the last reconciliation attempt.
+	// +optional
+	LastAttemptedPostRenderersDigest string `json:"lastAttemptedPostRenderersDigest,omitempty"`
```

Comment:
Please document this field under https://github.com/fluxcd/helm-controller/blob/main/docs/spec/v2/helmreleases.md#last-attempted-revision-digest